modules/barcode/src/decoder/upcean_decoder.cpp

```diff
@@ -0,0 +1,285 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+// Copyright (c) 2020-2021 darkliang wangberlinT Certseeds
+
+#include "../precomp.hpp"
+#include "upcean_decoder.hpp"
+#include <map>
+
+namespace cv {
+namespace barcode {
+
+static constexpr int DIVIDE_PART = 15;
+static constexpr int BIAS_PART = 2;
+//void UPCEANDecoder::drawDebugLine(Mat &debug_img, const Point2i &begin, const Point2i &end) const
+//{
+//    Result result;
+//    std::vector<uchar> middle;
```

Comment:
> `//`

Instead of using of multi-line comments, consider using of `#if 0` to disable debug parts in the code.